spanner/value.go

```diff
@@ -879,12 +879,7 @@ const (
 // supportsNull returns true for the Go types that can hold a null value from
 // Spanner.
 func (d decodableSpannerType) supportsNull() bool {
-	switch d {
```

Comment:
So anything can be null now?

Reply:
Nvm, I read the rest of this and this makes sense now.

Reply:
Yes. In our customized version of this, anything can accept null. This function only applies to the custom types; there was no such place before where it was made so obvious, but we added things such as this to the string decoding instead of returning an error:
```
if isNull {
	*p = ""
	break
}
```